ffxiv_visland/Gathering/GatherRouteDB.cs

```diff
@@ -39,6 +39,7 @@ public enum InteractionType
         EquipRecommendedGear = 10,
         ChatCommand = 11,
         NodeScan = 12,
+        SurveyNodeScan = 13
```

Comment:
I assume you just separated them to not get in the way of whatever you thought I was doing, but really it should be combined

Reply:
My thinking was there's cases where NodeScan covers the intended use, but SurveyNodeScan is overkill, but you're right - I think it would be better to have a SurveyNodeType that just does the basic NodeScan.